activity_calendar/models.py

```diff
@@ -565,7 +580,12 @@ def get_absolute_url(self):
 class Participant(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     activity_slot = models.ForeignKey(ActivitySlot, on_delete=models.CASCADE)
+    # Charfield for adding external users, this can only be done through admin.
+    guest_name = models.CharField(max_length=123, default='')
```

Comment:
Should probably add `blank=True` as empty values are allowed. The max length can also probably be lower